### DISCUSSION TITLE
Subtitles for videos?

### DISCUSSION THREAD

**Discussion by markbaumgarten** [zauberzeug/nicegui#2567] [Ideas / Feature Requests]
Is there a way to set subtitles (.srt file) on a video file?

I have created .srt files (and .vtt files) for my videos and would like my video to show subtitles. It works in VLC player, but is it possible using niceguis' video component?

**Comment by falkoschindler** [comment]
Thanks for the suggestion, @markbaumgarten!
At the moment this is not natively supported by NiceGUI. But since it [seems possible with HTML](https://stackoverflow.com/a/34848644/3419103), it should be possible to add vtt support for `ui.video`. Anyone up for a pull request? 🙂

**Reply by mxiaonian** [reply]
Hello, I have also had this requirement recently. I tried adding ui.element ui.add_head_html, but it doesn't seem to work. What should I do?

**Reply by falkoschindler** [reply]
@mxiaonian Can you, please, share what you tried so far? What exactly does not work?
